releasenotes/notes/fix-iqp-203557797d0d448e.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed the construction of Instantaneous Quantum Polynomial time (IQP) circuits
+    in :class:`~qiskit.circuit.library.IQP` and by :func:`~qiskit.circuit.library.iqp`:
+    T-gate is equivalent to P(pi/4) and not to P(pi/8).
```

Maybe this is a little clearer for users to understand (who didn't know the bug was a wrong angle)? 🙂 
```suggestion
    Fixed the construction of Instantaneous Quantum Polynomial time (IQP) circuits
    in :class:`~qiskit.circuit.library.IQP` and by :func:`~qiskit.circuit.library.iqp`:
    which was using powers of :math:`\sqrt{T}` gates instead of powers 
    of :math:`T` gates as single qubit operations.
```

---

This makes sense, I have reworded the note following your suggestion. I got rid of "single qubit operations" since H-gates are also single-qubit operations present in the circuit.